bin/distmap

```diff
@@ -346,7 +384,9 @@ sub check_mapper_integritiy {
 
 
 
-
+## TODO: is this really necessary??
+## TODO: the inputs should be populated before, and here there is no real "check"
+## TODO: in addition, this is overriding stuff that shouldn't be overriden
```

Comment:
Yes, this is a bit messy now. But again, let us not change this structure now.

Reply:
I just added the TODO to track the issues. I will open issues for all the TODOs here too.

Reply:
This is kind of related with #66, the same as the hash for parsing